File: code/src/test/scala/cern/colt/matrix/tint/impl/DenseLargeIntMatrix2DViewTest.scala
package cern.colt.matrix.tint.impl

//remove if not needed
import scala.collection.JavaConversions._

class DenseLargeIntMatrix2DViewTest(arg0: String) extends DenseLargeIntMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new DenseLargeIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new DenseLargeIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new DenseLargeIntMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tint/impl/DenseLargeIntMatrix2DViewTest.java
package cernx.colt.matrix.tint.impl;

public class DenseLargeIntMatrix2DViewTest extends DenseLargeIntMatrix2DTest {

    public DenseLargeIntMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseLargeIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseLargeIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseLargeIntMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }
}
